extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js

```diff
@@ -12,12 +12,11 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 	var logGroup = 'ext.wikia.adEngine.adInfoTrackerHelper';
 
 	function shouldHandleSlot(slot, enabledSlots) {
-		var dataGptDiv = slot.container.firstChild;
-
 		return (
 			enabledSlots[slot.name] &&
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)